src/app/screens/CreatePassword.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FormEvent } from 'react';
+
+import { createPassword, useDispatch } from '@store';
+import { passwordStrength } from 'check-password-strength';
+import { object, refine, string } from 'superstruct';
+import { FormError, useForm } from 'typed-react-form';
+
+import { getValidator } from '@app/utils';
+import warning from '@assets/icons/circle-warning.svg';
+import {
+  Body,
+  Box,
+  Button,
+  Flex,
+  FormInput,
+  Heading,
+  Image,
+  Label,
+  PanelBottom
+} from '@components';
+import { getKBHelpArticle, KB_HELP_ARTICLE } from '@config/helpArticles';
+import { translate, translateRaw } from '@translations';
+import { PasswordStrength } from '@types';
+
+const CREATE_PASSWORD_STRUCT = object({
+  password: refine(string(), 'Strong password', (value) => {
+    if (passwordStrength(value).id === PasswordStrength.STRONG) {
+      return true;
+    }
+
+    return translateRaw('PASSWORD_TOO_WEAK');
+  }),
+  passwordConfirmation: refine(string(), 'Equal to password', (value, context) => {
+    return value === context.branch[0].password;
+  })
+});
+
+export const CreatePassword = () => {
+  const dispatch = useDispatch();
+  const form = useForm(
+    {
+      password: '',
+      passwordConfirmation: ''
+    },
+    getValidator(CREATE_PASSWORD_STRUCT)
+  );
+
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    await form.validate();
+    if (form.error) {
+      return;
+    }
+
+    dispatch(createPassword(form.values.password));
+  };
+
+  return (
+    <Box>
+      <Heading fontSize="24px" lineHeight="150%" mt="24px" mb="8px" textAlign="center">
```

Comment:
FYI you can use margin `1` `2` `3` etc. which uses theme values, that way it is standardized across the app